markdown/setup/dbvisualizer-driver-setup.md

```diff
@@ -0,0 +1,42 @@
+### DbVisualizer 
+[Link to product webpage](https://www.dbvis.com/download/).
+
+#### Adding the Amazon Timestream JDBC Driver
+1. [Download](https://github.com/awslabs/amazon-timestream-driver-jdbc/releases/latest) the Timestream JDBC driver shaded JAR file (e.g., `amazon-timestream-jdbc-2.0.0-shaded.jar`)
+2. Start the DbVisualizer application and navigate to the menu path: **Tools > Driver Manager...**. In the popped up window, click the "Custom" link on the right side.
```

Comment:
Shall the user click on the "+" icon to add the driver?

Reply:
No. The "+" is only to add a driver from a template driver. We do not have a template driver so user could only add through "Custom" link.

Reply:
Alternative option:
<img width="282" alt="image" src="https://user-images.githubusercontent.com/7035336/206814445-e796ea43-bb2f-449b-8f32-ca4f2e67c767.png">

Reply:
Ok, let me add it.